dev/source/docs/ros2.rst

```diff
@@ -17,49 +17,59 @@ Prerequisites
 
-You are about to create a new `ROS 2 workspace <https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace.html#id4>`__.
-This page assumes that your workspace is named `ardu_ws` in your home directory, but feel free to adjust to your preferred location.
-
-Before anything else, make sure that you have `sourced your ROS 2 environment <https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files>`__
-and check if it is `configured correctly <https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#check-environment-variables>`__.
-
-Finally, ensure you have `set up your ArduPilot build environment <https://ardupilot.org/dev/docs/building-the-code.html#setting-up-the-build-environment>`__.
+This document suggests creating a new ROS 2 workspace from scratch, named `ardu_ws`, located in your home folder. If you already have a working ROS 2 workspace, feel free to adapt the instructions to your needs.
 
 Installation (Ubuntu)
 =====================
 
-To make installation easy, we will clone the required repositories using `vcs` and a `ros2.repos` files:
+To make installation easy, we will clone the required repositories using `vcs` and a `.repos` files:
 
-.. code-block:: bash
+.. tabs::
 
-    mkdir -p ~/ardu_ws/src
-    cd ~/ardu_ws
-    vcs import --recursive --input  https://raw.githubusercontent.com/ArduPilot/ardupilot/master/Tools/ros2/ros2.repos src
+    .. tab:: Jazzy
+
+        .. code-block:: bash
+
+            mkdir -p ~/ardu_ws/src
+            cd ~/ardu_ws
+            vcs import --recursive --input  https://raw.githubusercontent.com/ArduPilot/ardupilot/master/Tools/ros2/jazzy.repos src
+            
+    .. tab:: Humble
+
+        .. code-block:: bash
+
+            mkdir -p ~/ardu_ws/src
+            cd ~/ardu_ws
+            vcs import --recursive --input  https://raw.githubusercontent.com/ArduPilot/ardupilot/master/Tools/ros2/humble.repos src
 
 This will take a few minutes to clone all the repositories your first time.
 
-Now update all dependencies:
+.. warning::
+    The `ardupilot` repository will be cloned in the `~/ardu_ws/src` folder during this process. Before proceeding, `set up your ArduPilot build environment <https://ardupilot.org/dev/docs/building-the-code.html#setting-up-the-build-environment>`__ inside the ROS 2 workspace.
+    If you already have a working ArduPilot build environment, you may move it inside the workspace instead, but be sure to change all script references too. It might be a good idea to check your `.bashrc`.
+
+Now update all dependencies using `rosdep`:
 
 .. code-block:: bash
 
     cd ~/ardu_ws
     sudo apt update
     rosdep update
-    source /opt/ros/humble/setup.bash
```

Comment:
This might be unnecessary, the instructions assume that the ROS environment is setup and this is explicitly stated at the start

Reply:
It is not required for people to add this to your ~/.bashrc. I have tried to make the commands as foolproof as possible, and becuase of that, our support load has decreased a lot.